test/suites/clustering.sh

```diff
@@ -245,7 +245,7 @@ test_clustering_membership() {
   LXD_DIR="${LXD_THREE_DIR}" lxc cluster list
   LXD_DIR="${LXD_TWO_DIR}" lxc cluster show node1 | grep -xF -- '- database-leader'
   [ "$(LXD_DIR="${LXD_THREE_DIR}" lxc cluster list | grep -Fc "database-standby")" = "2" ]
-  [ "$(LXD_DIR="${LXD_FIVE_DIR}" lxc cluster list | grep -Fc "database ")" = "3" ]
+  [ "$(LXD_DIR="${LXD_FIVE_DIR}" lxc cluster list | grep -Fc "database-voter")" = "2" ]
```

Comment:
I've been thinking that `grep -wFc` would be nice as `-w` makes sure it doesn't match on partial role names. Granted this would have made even more sense before to avoid needing the trailing ` ` in the pattern.

Who knows, maybe we'll get a `database-voter-backup` or `database-voter-denied` role in the future?

The other `grep` patterns use `-x` so should be safe from this.